webui/src/data/server.js

```diff
@@ -529,7 +554,7 @@ class ServerCache {
     }
 
     getRecord(id) {
-        this.getters.record.get(id).fetch();
+        this.getters.record.get(id).forceFetch();
```

Comment:
Can we avoid this change? It would hurt the performance when re-displaying records.

Reply:
@emanueldima Right, I removed this. I added it in an attempt to get a fresh version of the record page because the cached one was outdated. Navigating to a directly older version rendered the page as when that was the latest one. The versions are shown correctly now but I can't get around this annoying rerender of the dropdown list and the create button. 
![versioning_lag](https://user-images.githubusercontent.com/1566093/32494891-c29a8c8c-c3c3-11e7-8607-b5a42e545d05.gif)